### PR TITLE
Add index not found message to each tab of the fim module

### LINKED ISSUE #6325: FIM module - Index not found message
## Description

In the FIM module, when an index is not found, the dashboard loads but without data, the same happens in the inventory and events tab of the same module.

### What we want:

We want that if the index is not found the dashboard does not load and displays an index not found message, the same behaviour we want to happen in the inventory and events tab of the same module.

## Tasks
The steps that have to be completed in order to close the issue.

- [x] Reuse components from other application modules to achieve the desired behavior.

## Additional information
Add here any additional information relevant to the issue or that will help to close it: chunks of code, logs, additional error messages, related issues and so on.

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

In the FIM module, when an index is not found, the dashboard loads but without data, the same happens in the inventory and events tab of the same module.

What we want:
We want that if the index is not found the dashboard does not load and displays an index not found message, the same behaviour we want to happen in the inventory and events tab of the same module.

image

image

image
 
### Issues Resolved

#6325 

### Evidence

image

image

image

### Test

Go to the FIM module and check that each tab looks like the images attached in this PR.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [x] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — DISMISSED** [review]
## CR: :green_circle: 

## Test UI :green_circle: 

### Evidences
#### Dashboard

image


### Inventory

image

#### Events

image

**Comment by lucianogorza** [comment]
## CR 🟡 
## Test 🟢 
image
image
image

**Review by lucianogorza — COMMENTED** [review]
I suggest making a few changes to the code to enhance readability and minimize errors.

File: plugins/main/public/components/overview/fim/inventory/inventory.tsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ import { withErrorBoundary } from '../../../common/hocs';
 import { HitsCounter } from '../../../../kibana-integrations/discover/application/components/hits_counter/hits_counter';
 import { formatNumWithCommas } from '../../../../kibana-integrations/discover/application/helpers';
 import { useAppConfig } from '../../../common/hooks';
+import { WAZUH_INDEX_TYPE_FIM } from '../../../../../common/constants';
+import useCheckIndexFields from '../../vulnerabilities/common/hooks/useCheckIndexFields';
```
Comment (yenienserrano) [inline plugins/main/public/components/overview/fim/inventory/inventory.tsx:45]: Perhaps the vulnerability files should be moved if they have not already been moved.

**Pull request closed without merging**